### ISSUE TITLE
wrongthink.net

### ISSUE THREAD

**Issue by zQueal** [keybase/keybase-issues#2950]
@1337god @LouiseMensch

Opened this thread to keep discussion out of #518 

```html
<p dir="auto">Verifying myself: I am 1337god on Keybase.io. 52Rp68Br827ru2xryZ88gKd355_62u22FxZQd / <a href="https://keybase.pub/1337god/Verification/wrongthink.txt" target="_blank" class="hash" rel="nofollow">https://keybase.pub/1337god/Ve....rification/wrongthin</a></p>

```

For reference, it seems that the platform truncates urls which are beyond a preset limit.

![xu0ao](https://cloud.githubusercontent.com/assets/239193/24829125/3290972a-1c3a-11e7-8d59-c0e9489ecc20.png)

This could make it difficult for the web scraper to identify a valid proof--but I'm not sure.

**Comment by 1337god** [comment]
Oh ok sorry

**Comment by 1337god** [comment]
I'll try check if I can make it filter some urls so it doesn't truncate them, for example keybase.io & keybase.pub

**Comment by jcgruenhage** [comment]
Wouldn't it be better to make this a more general verification? Since wrongthink.net is basically a GNU social instance (isn't it?), one should be able to craft a general OStatus verification instead, that works with all GNU social and mastodon instances.

**Comment by 1337god** [comment]
WrongThink is no longer a GNU Social Instance due to problems with the software. And we had investment from the #AnimeRight to proceed with the project we now use WoWonder.

**Comment by jcgruenhage** [comment]
That thing doesn't federate, I guess? If it used OStatus, it might have been more interesting. In addition to that, if it is an unfederated, closed source platform, what exactly is the advantage over twitter? Another alternative to GNUSocial would have been to use mastodon, which also implements OStatud

**Comment by 1337god** [comment]
It's more of a Facebook Alternative with a mix of Twitter.

**Comment by 1337god** [comment]
and we will add Blogs, Pages (likes), groups, profiles (follow), user-based ads (no advertisement from companies), it's a non-profit project.

**Comment by jcgruenhage** [comment]
Still, wouldn't it be better for the community (especially as a non-profit) to use open source software for that, that federates with people hosting their own instances of said software? Another walled garden is just not the solution :D

**Comment by 1337god** [comment]
Tried did not work.

**Comment by jcgruenhage** [comment]
Mastodon? I know you had problems with gnu-social, but out of these problems, a group of devs/users created mastodon, which looks quite a bit nicer ^^

But anyway, do as you wish. If you see WoWonder a better fit, go ahead and use that :D

**Comment by 1337god** [comment]
Mastodon, I'll use it for Encyclopedia Dramatica Instance, but still it will get censored by all other mastodon instances, they censor everything that has a political opinion different than them. they have been caught blocking instances bc of what they post.

**Comment by jcgruenhage** [comment]
Well, mastodon.social blocks them, but everybody that hosts their own instance can choose for themselves what they block and what they let through. I would argue that most content federates to other instances is still better than no federation at all :P

For example, I would not block other instances at all on my instance (once I've set that up), and therefore I could follow people on wrongthink.net, if they used something OStatus compliant, how it is now, that is impossible ;)

**Comment by 1337god** [comment]
If it was just mastodon.social, some people are blocking others in order to not be blocked by others who block those kind of instances.

**Comment by 1337god** [comment]
And I'm working on getting OStatus on WrongThink, it will take a long time but I'll try.

**Comment by 1337god** [comment]
But ur doing good, don't block. Also what's your instance btw?

**Comment by jcgruenhage** [comment]
Non-existent, for now, mastodon.social first needs LDAP support, since I won't create a new user db for that, but that will be reachable under https://gruenhage.xyz/, like my main matrix server ^^

**Comment by 1337god** [comment]
For what I heard but it's not confirmed Allegations, if it's really true, it's very bad yeah.

**Comment by 1337god** [comment]
And some instances will block some kind of speech and limit types of uploads in order to not get blocked by others too.

**Comment by 1337god** [comment]
Also I might join yours once it's up.

**Comment by zQueal** [comment]
> Posts from users on external instances are "federated" into the local one, i.e. if user1@mastodon1 follows user2@gnusocial2, any posts user2@gnusocial2 makes appear in both user1@mastodon1's Home feed and the public timeline on the mastodon1 server. Mastodon server administrators have some control over this and can exclude users' posts from appearing on the public timeline; post privacy settings from users on Mastodon instances also affect this, see below in the Toot Privacy section.

As far as I understand this issue the selective federation only applies to the single server. In the example above, if user1 follows user2, but user2 is "blocked" on mastodon1 (user1's server), then user2's tweets don't reach the mastodon1's federated public timeline, but user1 will still be able to view their content. If that's wrong then it might be pretty hairy to implement proofs, especially if Keybase is blocked from a specific server and it propagates outward or something.

**Comment by jcgruenhage** [comment]
This might clear things up a bit: https://hackernoon.com/mastodon-is-dead-in-the-water-888c10e8abb1

**Comment by 1337god** [comment]
@zQueal That only applies to Mastodon Instances, on GNU/Social instances they will never block anyone. Not even Qvitter does that, and they are pretty SJW.
